doc/source/jupyterhub/customizing/user-resources.md

```diff
@@ -202,3 +202,40 @@ az aks show --name <YOUR-CLUSTER-NAME> --resource-group <YOUR-RESOURCE-GROUP>
 
 It may take some time for the new cluster nodes to be ready.
 You can use `kubectl get node` to report the current `Ready/NotReady` status of all nodes in the cluster.
+
+### Amazon Web Services Elastic Kubernetes Service (EKS)
+
+To scale an existing nodegroup using the `eksctl` command line tools:
```

Comment:
Is this a valid step for anyone having an EKS cluster, no matter if it was setup with `kops` or `eksctl` initially? If not, I think it should be clarified that this is scoped specifically for AWS EKS clusters setup with `eksctl`.

Reply:
Good point. This is EKS specific and doesn't apply to either `kops` or a "manual" k8s installation in AWS. I'll clarify.